extractor/extraction_containers.cpp

```diff
@@ -288,20 +293,20 @@ void ExtractionContainers::PrepareEdges(lua_State *segment_state)
             continue;
         }
 
-        if (edge_iterator->result.target < node_iterator->node_id)
+        if (edge_iterator->result.osm_target_id < node_iterator->node_id)
         {
-            SimpleLogger().Write(LogLevel::logWARNING) << "Found invalid node reference " << edge_iterator->result.target;
+            SimpleLogger().Write(LogLevel::logWARNING) << "Found invalid node reference " << NodeID64_to_uint64_t(edge_iterator->result.osm_target_id);
```

Comment:
Where is `NodeID64_to_uint64_t()` defined? And wouldn't it be better to implement a conversion operator so we can write `uint64_t(...)`?